### DISCUSSION TITLE
第一次播放正常，部分安卓机型第二次播放非常卡

### DISCUSSION THREAD

**Discussion by 2574161467** [Tencent/libpag#1837] [Q&A] [answered]
第一次播放正常，部分安卓机型(安卓10)第二次播放非常卡

**Comment by 2574161467** [comment]
目前将pag放在了进入页面的时候加载 第一次播放完毕之后将动画销毁后重新创建

**Comment by kevingpqi123** [comment]
针对此类问题，建议先在 pag 的 官方 demo 中测试，如果 官方 demo 有问题，我们基于官方 demo 进行排查，如果没有问题，那就排查你们的实现

**Reply by 2574161467** [reply]
使用同时创建多个动画的官方demo 替换素材后同时播放多个就会很卡 只播放一个就还好 这样能优化吗

**Reply by 2574161467** [reply]
这种只能从素材方面优化吗 目前我们的动画素材使用的是矢量与BMP混合导出的方式

**Reply by domchen** [reply, marked as answer]
请参考这里进行素材性能的优化：https://pag.art/docs/optimization.html
